src/hooks/useFormJson.tsx

```diff
@@ -130,3 +136,76 @@ function setEncounterType(formJson: OHRIFormSchema): void {
     delete formJson.encounter;
   }
 }
+
+/**
+ * Functions to support reusable Form Components
+ */
+function getReferencedForms(formJson: OHRIFormSchema): Array<ReferencedForm> {
+  const referencedForms: Array<any> = formJson.referencedForms;
```

Comment:
what happens when `formJson` is `undefined` ?

Reply:
This has been handled with the typings and validation on the call